File: Java/UberVan.java
package Java;


import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car{
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    public UberVan(String license, Account driver){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    
}
}

File: Python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    setsMaterial = []

    def __init__(self,license, driver, typeCarAccepted,seatsMaterial ):
        super().__init__(license,driver)
        self.typeCarAccepted = typeCarAccepted
        self.setsMaterial = seatsMaterial